apps/ocpp/admin/common.py

```
@@ -85,3 +85,36 @@ def log_view(self, request, object_id):
+                "No non-deleted simulators were selected.",
+                level=messages.ERROR,
+            )
+            return
+        default_simulator = selected[0]
+        if len(selected) > 1:
+            self.message_user(
+                request,
+                f"Multiple simulators selected; {default_simulator.name} was set as default.",
+                level=messages.WARNING,
+            )
+        with transaction.atomic():
```

Ruff (0.14.14)

[error] Undefined name `transaction`

(F821)